src/main/java/com/dragonballs/security/WebSecurityConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.dragonballs.security;
+
+import com.dragonballs.services.authentication.JwtAuthorizationTokenFilter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@EnableWebSecurity
+@Configuration
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Autowired
+    JwtAuthorizationTokenFilter authenticationTokenFilter;
+
+    @Override
+    protected void configure(HttpSecurity httpSecurity) throws Exception {
+        //http.csrf().disable();
+        httpSecurity
+                // we don't need CSRF because our token is invulnerable
```

Comment:
What do you mean? :D 
The only reason I could agree that csrf is not necessary is testing env.